tuna/miopen/worker/fin_eval.py

```diff
@@ -54,10 +54,12 @@ def __init__(self, **kwargs):
   def get_job(self, find_state, set_state, imply_end):
     """Polling to see if job available"""
     self.logger.info('find job: %s', find_state)
+
     if not super().get_job(find_state, set_state, imply_end):
       with self.bar_lock:
         self.num_procs.value -= 1
       return False
```

Comment:
This should be evaluated after we've finished checking for all job times. After checking "compiled" and "new". Only when there are no more jobs.